resources/views/errors/layout.blade.php

```diff
@@ -15,10 +13,10 @@
                 margin: 0;
                 padding: 0;
                 width: 100%;
-                color: #B0BEC5;
+                color: #999999;
                 display: table;
                 font-weight: 100;
-                font-family: 'Lato';
+                font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
```

Comment:
I suggest adding 'Lato Light' at first so it's quite the same as before on systems where the font exist.
```suggestion
                font-family: "Lato Light", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
```